packages/botonic-react/src/webchat/webchat.jsx

```diff
@@ -190,10 +190,22 @@ export const Webchat = forwardRef((props, ref) => {
   const theme = merge(webchatState.theme, props.theme)
   const { initialSession, initialDevSettings, onStateChange } = props
   const isOnline = useNetwork()
-  const [botonicState, saveState, deleteState] = useLocalStorage('botonicState')
+  const getThemeProperty = _getThemeProperty(theme)
+
+  const getStorage = () => {
+    const storage = getThemeProperty('storage', props.storage)
+    if (storage === undefined) return localStorage
```

Comment:
be aware that "null === undefined".
Which means that the unit test where you call  
`TestRenderer.create(<Webchat storage={null} />)
`
might break in the future if getThemeProperty implementation is changed.
I think it's better to use == undefined

Reply:
In case `storage == null` it should return null and then the botonicState will not be stored anywhere. Otherwise, if `storage == undefined` the botonicState will be stored in the localStorage. If I use `null == undefined`, it does not work as desired.